.env.example

```diff
@@ -1,3 +1,3 @@
-PORT=5000
+PORT=3000
```

Comment:
Any reason why you changed this from 5000 to 3000?

Reply:
At a point, I felt it will be easier to tell a user to use the PORT in `./src/index.js` file.